tests/flow/test_concurrent_query.py

```diff
@@ -8,31 +8,16 @@
 CLIENT_COUNT = 16                   # Number of concurrent connections.
 people = ["Roi", "Alon", "Ailon", "Boaz", "Tal", "Omri", "Ori"]
 
-def query_aggregate(query):
-    env = Env(decodeResponses=True)
-    conn = env.getConnection()
-    graph = Graph(GRAPH_ID, conn)
-    return graph.query(query)
-
-def query_neighbors(query):
-    env = Env(decodeResponses=True)
-    conn = env.getConnection()
-    graph = Graph(GRAPH_ID, conn)
-    return graph.query(query)
-
-def query_write(query):
-    env = Env(decodeResponses=True)
-    conn = env.getConnection()
-    graph = Graph(GRAPH_ID, conn)
-    return graph.query(query)
-
 def thread_run_query(query):
     env = Env(decodeResponses=True)
     conn = env.getConnection()
     graph = Graph(GRAPH_ID, conn)
 
     try:
-        return graph.query(query)
+        result = graph.query(query)
```

Comment:
Would it be possible to place a `barrier` between all processes in the same pool just before executing the query?

Reply:
done